src/dataviews/histogram-dataview/histogram-data-model.js

```diff
@@ -120,5 +124,11 @@ module.exports = Model.extend({
     }
 
     return parsedData;
+  },
+
+  _getCurrentOffset: function () {
```

Comment:
It's a function only used in this module, to form the URL with the saved offset or the local timezone. It's not needed outta here.